src/Activity.js

```diff
@@ -102,6 +90,4 @@ class Activity {
   }
 }
 
-
-
-export default Activity;
+export default Activity;
```

Comment:
This looks way cleaner now

Reply:
Awesome, thanks. We might even be able to remove a few more methods that find information that the spec doesn't ask for!